vue.config.js

```diff
@@ -18,5 +18,9 @@ module.exports = {
 
       config.devtool('inline-cheap-module-source-map')
     }
+
+    // set up aliases for mock services, used when the offline mode is used
+    const suiteService = process.env.NODE_ENV === 'offline' ? '@/services/mock/suite.service' : '@/services/suite.service';
```

Comment:
I wonder if you can break up long lines with ternary operator this way?

```javascript
const suiteService = process.env.NODE_ENV === 'offline'
    ? '@/services/mock/suite.service'
    : '@/services/suite.service';
```

Reply:
Yup! Worked with no issues.